### DISCUSSION TITLE
posts/%E6%95%99%E7%A8%8B/archconfiguration/

### DISCUSSION THREAD

**Discussion by giscus** [LingXi9374/LingXi9374.github.io#12] [General]
# posts/%E6%95%99%E7%A8%8B/archconfiguration/

讲述安装Arch Linux后，需要干什么

https://lingxi9374.github.io/posts/%E6%95%99%E7%A8%8B/archconfiguration/

**Comment by hatateaya** [comment]
资瓷

**Comment by LingXi9374** [comment]
图片发送测试
![pAXikBn.png](https://s21.ax1x.com/2024/12/21/pAXikBn.png)

**Reply by LingXi9374** [reply]
回复发送图片测试

![pAzziIx.png](https://s21.ax1x.com/2025/01/01/pAzziIx.png)

**Comment by PermeationLoop** [comment]
資瓷，但是我已經是 Manjaro 的形狀了，，，

既能能享受 GUI 的便利，轉身又能掏空 AUR 的軟件生態，兩頭通吃，豈不美哉，，，

俺相信 Manjaro 是最好的 Arch

**Reply by LingXi9374** [reply]
Manjaro *开箱即用* 是好的，但它对aur软件包的`兼容性不如Arch`，很多时候aur来源的软件在Arch上正常运行，反而放到manjaro上就有一堆bug与崩溃😂
Arch社区对manjaro的问题不予处置也是因为 *系统环境不同* （manjaro好像改了很多东西，并且定位也与Arch不同，用于生产环境，更新周期也比Arch慢），解决起来很麻烦
在我看来`Endeavour OS`是最好的开箱即用Arch发行版，没有改太多东西，等于Arch加了个Endeavour的少量套件与桌面主题，具体来说各取所需得了，manjaro的出现也有它存在的意义

**Reply by PermeationLoop** [reply]
Manjaro 的推出是在 2011 年，Endeavour 是在 2019。而我裝 Linux 筆電是在 2020 年，當時 Endeavour 還不出名，再加上 Ubuntu 上驅動不夠新，有些硬件跑不起來，所以那時我纔選擇了 Manjaro。

至於你說得 AUR 軟件包的兼容性問題，我沒有遇到過。如果是依賴版本導致的，那麼包管理器應該能通過自動編譯解決纔對。妳能否舉一點例子給我看看？

PS：Arch 這個花園太大，裏面花的品種也很多。然而我發現，前來賞花的人並不會專注與自己的花，而是直接會跑到別人邊上說「妳這朵花不行！」。具有相同風氣的圈子，應該只剩下二遊了，屬實難繃。

https://github.com/user-attachments/assets/9e3de3bb-3186-42d0-85c3-b4e6b27f762a
